app/assets/stylesheets/title.scss

```diff
@@ -0,0 +1,22 @@
+$elegant: #f0efee;
+
+h1 {
+  font-family: "Avant Garde", Avantgarde, "Century Gothic", CenturyGothic, "AppleGothic", sans-serif;
+  font-size: 30px;
+  text-align: center;
+  text-transform: uppercase;
+  text-rendering: optimizeLegibility;
+
+  &.elegantshadow {
+    color: #131313;
+    background-color: $elegant;
+    letter-spacing: .15em;
+    text-shadow:
+      1px -1px 0 #767676,
```

Comment:
Color literals like `#e2e0df` should only be used in variable declarations; they should be referred to via variable everywhere else.